### ISSUE TITLE
Introduction to R episode needs more exercises

### ISSUE THREAD

**Issue by bioinfowlad** [datacarpentry/r-socialsci#343]
The Subsetting Vectors section of the Introduction to R episode <https://datacarpentry.org/r-socialsci/01-intro-to-r/index.html> needs an exercise or 2: it is introducing two concepts (subsets of data, logical vectors) and combining them in a way that is counterintuitive for some.

**Comment by juanfung** [comment]
@bioinfowlad Thanks for the feedback. Would you be willing to submit a PR to add a couple of exercises?

**Comment by bioinfowlad** [comment]
Yes, i'm starting to submit PRs to these episodes 2day :)

**Comment by bioinfowlad** [comment]
Yes, but I haven't scheduled it. I can close the issue if that helps.

**Comment by juanfung** [comment]
@bioinfowlad If you're planning to, we can keep it open. If not, no problem - I can add a couple of exercies.

**Comment by juanfung** [comment]
@bioinfowlad closing for now. please reopen if and when you are ready to add!
